server/controller/src/main/java/ai/starwhale/mlops/domain/project/ProjectManager.java

```diff
@@ -37,6 +37,8 @@
 @Service
 public class ProjectManager implements ProjectAccessor {
 
+    public static final String PROJECT_SEPERATOR = ":";
```

Comment:
will the console shows project under namespace like `ns:project`. Users may copy the project name directlly to do upload or download